tesseract_motion_planners/ompl/include/tesseract_motion_planners/ompl/profile/ompl_profile.h

```diff
@@ -92,6 +85,13 @@ class OMPLPlanProfile : public Profile
                     const tesseract_common::ManipulatorInfo& composite_mi,
                     const std::shared_ptr<const tesseract_environment::Environment>& env) const = 0;
 
+  /**
+   * @brief Convert an OMPL defined path into a composite instruction that can be returned by the planner
+   */
+  virtual CompositeInstruction convertPath(const ompl::geometric::PathGeometric& path,
+                                           const tesseract_common::ManipulatorInfo& composite_mi,
+                                           const std::shared_ptr<const tesseract_environment::Environment>& env) const = 0;
```

Comment:
To mirror what is currently being done in [`assignTrajectory`](https://github.com/tesseract-robotics/tesseract_planning/blob/master/tesseract_motion_planners/ompl/src/ompl_motion_planner.cpp#L205), do we also need to pass in the UUIDs of the start and end waypoints or those waypoints themselves?

Reply:
I  like this change. I was using the minimum needed but you could also pass the waypoint or instruction.